Первичная верстка

GrushchakArt opened the pull request.

GrushchakArt committed:
Первичная верстка

lookatmycat commented on AdventureMain.css:
```diff
@@ -0,0 +1,123 @@
+    width: 30%;
+    height: 47%;
+}
+.PlotDesctiption{
```
имена классов в kebab case

lookatmycat commented on AdventureMain.css:
```diff
@@ -0,0 +1,123 @@
+}
+.action1:hover, .action1:active{
+    background-color: #d9950d;
+    background-color: rgba(217, 149, 13, 0.4);
```
отменяет действие предыдущей строки

lookatmycat commented on AdventureMain.css:
```diff
@@ -0,0 +1,123 @@
+    border-color: #d9950d;
+    border-color: rgba(217, 149, 13, 0.4);
+    transition: background-color 0.3s ease-in,
+    border-color 0.3s ease-in;
```
опечатка

lookatmycat commented on AdventureMain.css:
```diff
@@ -0,0 +1,123 @@
+    background-color: rgba(217, 149, 13, 0.4);
+    border-color: #d9950d;
+    border-color: rgba(217, 149, 13, 0.4);
+    transition: background-color 0.3s ease-in,
```
запятая?

lookatmycat commented on AdventureMain.css:
```diff
@@ -0,0 +1,123 @@
+    background-color: #d9950d;
+    background-color: rgba(217, 149, 13, 0.4);
+    border-color: #d9950d;
+    border-color: rgba(217, 149, 13, 0.4);
```
отменяет действие предыдущей строки

lookatmycat reviewed (commented):
все названия файлов в camel case, в данном случае 
такие index.html, main.css, main.js

GrushchakArt committed:
Первичная верстка

GrushchakArt committed:
Первичная верстка

GrushchakArt committed:
Первичная верстка

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.
Смена кнопок с тега input на тег button.

Первичная обработка клика.

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.
Смена кнопок с тега input на тег button.
Первичная обработка клика.

Добавил тестовый набор текстовых блоков и действий.

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.
Смена кнопок с тега input на тег button.
Первичная обработка клика.

Добавил тестовый набор текстовых блоков и действий.

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.
Смена кнопок с тега input на тег button.
Первичная обработка клика.

Добавил тестовый набор текстовых блоков и действий.

GrushchakArt committed:
Первичная верстка
Поправил по цвету текста в блоках.
Смена классов на кебаб-кейс.
Перенос подключения js в конец body.
Рефакторинг css.
Смена кнопок с тега input на тег button.
Первичная обработка клика.

Добавил тестовый набор текстовых блоков и действий.

GrushchakArt committed:
Первичная верстка                  19.10.18
Поправил по цвету текста в блоках. 19.10.18
Смена классов на кебаб-кейс.       19.10.18
Перенос подключения js
 в конец body.                     19.10.18
Рефакторинг css.                   19.10.18
Смена кнопок с тега input на
 тег button.                       19.10.18
Первичная обработка клика.         19.10.18
Добавил тестовый набор текстовых
 блоков и действий.                19.10.18
Добавил обьекты: персонаж, монстр,
 оружие, броня.                    22.10.18

GrushchakArt committed:
Первичная верстка                  19.10.18
Поправил по цвету текста в блоках. 19.10.18
Смена классов на кебаб-кейс.       19.10.18
Перенос подключения js
 в конец body.                     19.10.18
Рефакторинг css.                   19.10.18
Смена кнопок с тега input на
 тег button.                       19.10.18
Первичная обработка клика.         19.10.18
Добавил тестовый набор текстовых
 блоков и действий.                19.10.18

Добавил обьекты: персонаж, монстр,
 оружие, броня.                    22.10.18
Добавил цикл в renderState         22.10.18

GrushchakArt committed:
Первичная верстка                  19.10.18
Поправил по цвету текста в блоках. 19.10.18
Смена классов на кебаб-кейс.       19.10.18
Перенос подключения js
 в конец body.                     19.10.18
Рефакторинг css.                   19.10.18
Смена кнопок с тега input на
 тег button.                       19.10.18
Первичная обработка клика.         19.10.18
Добавил тестовый набор текстовых
 блоков и действий.                19.10.18

Добавил обьекты: персонаж, монстр,
 оружие, броня.                    22.10.18
Добавил цикл в renderState         22.10.18
